### PR TITLE
OSGeo4W BAT files review

### PR DESCRIPTION
**Pull request by landam**
List of issues:

* `env.bat` doesn't need to define `PYTHONHOME` since it's already defined by `py3_env.bat`
* `py3_env.bat` doesn't need to be called from `grass.bat` since it's called by `env.bat`
* `package.sh`: development versions executable do not use any postfix, currently eg. `grass79`

### CODE REVIEW

File: mswindows/osgeo4w/package.sh [hunk 1]
```diff
@@ -113,11 +113,7 @@ read PATCH <&3
 export VERSION=${MAJOR}.${MINOR}.${PATCH}
 export POSTFIX=${MAJOR}${MINOR}
 
-if [[ "$PATCH" == *svn* ]] ; then
-    GRASS_EXECUTABLE=grass${MAJOR}${MINOR}svn
-else
-    GRASS_EXECUTABLE=grass${MAJOR}${MINOR}
-fi
+GRASS_EXECUTABLE=grass${MAJOR}${MINOR}
```
Comment (hellik) [inline mswindows/osgeo4w/package.sh]: this seems not to be related to _OSGeo4W BAT files review_ ? please sort it out from this PR.

File: mswindows/osgeo4w/env.bat.tmpl [hunk 2]
```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
...
-set PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37
-
 set GRASS_PROJSHARE=%OSGEO4W_ROOT%\share\proj
 
+set FONTCONFIG_FILE=%GISBASE%\etc\fonts.conf
+
+REM
+REM GDAL-related
+REM
 set PROJ_LIB=%OSGEO4W_ROOT%\share\proj
```
Comment (hellik) [inline mswindows/osgeo4w/env.bat.tmpl]: it's defined in _C:\OSGeo4W64\etc\ini\proj.bat_

`SET PROJ_LIB=%OSGEO4W_ROOT%\share\proj`

also removing?

File: mswindows/osgeo4w/env.bat.tmpl [hunk 3]
```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
...
 set GDAL_DATA=%OSGEO4W_ROOT%\share\gdal
```
Comment (hellik) [inline mswindows/osgeo4w/env.bat.tmpl]: it'd defined in _C:\OSGeo4W64\etc\ini\gdal.bat_

`SET GDAL_DATA=%OSGEO4W_ROOT%\share\gdal`

also removing?

File: mswindows/osgeo4w/env.bat.tmpl [hunk 4]
```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
...
 set GEOTIFF_CSV=%OSGEO4W_ROOT%\share\epsg_csv
```
Comment (hellik) [inline mswindows/osgeo4w/env.bat.tmpl]: it's defined in _C:\OSGeo4W64\etc\ini\libgeotiff.bat_

`SET GEOTIFF_CSV=%OSGEO4W_ROOT%\share\epsg_csv`

also removing?

File: mswindows/osgeo4w/env.bat.tmpl [hunk 5]
```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
 REM set GRASS_SH=%OSGEO4W_ROOT%\apps\msys\bin\sh.exe
 
 set GRASS_PYTHON=%OSGEO4W_ROOT%\bin\python3.exe
-set PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37
```
Comment (hellik) [inline mswindows/osgeo4w/env.bat.tmpl]: yes, it's defined in `C:\OSGeo4W64\bin\py3_env.bat`

`SET PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37`

File: mswindows/osgeo4w/grass.bat.tmpl [hunk 6]
```diff
@@ -9,7 +9,6 @@ rem
 rem Set environmental variables
 rem
 call "%~dp0\o4w_env.bat"
-call py3_env.bat
```
Comment (hellik) [inline mswindows/osgeo4w/grass.bat.tmpl]: yes it is callled in _C:\OSGeo4W64\apps\grass\grass78\etc\env.bat_
```

REM
REM Environmental variables for GRASS OSGeo4W installer
REM

call %OSGEO4W_ROOT%\bin\py3_env.bat


```

File: mswindows/osgeo4w/package.sh (hunk 1 above)
Comment (landam) [reply mswindows/osgeo4w/package.sh]: See #445

File: mswindows/osgeo4w/env.bat.tmpl (hunk 2 above)
Comment (landam) [reply mswindows/osgeo4w/env.bat.tmpl]: Right, `proj.bat` is called by `o4w_env.bat`. Removed by 7b3bb7a76

File: mswindows/osgeo4w/package.sh (hunk 1 above)
Comment (landam) [reply mswindows/osgeo4w/package.sh]: See #445

File: mswindows/osgeo4w/env.bat.tmpl (hunk 3 above)
Comment (landam) [reply mswindows/osgeo4w/env.bat.tmpl]: Right, `gdal.bat` is called by `o4w_env.bat`. Removed by c2fd0e3c5

File: mswindows/osgeo4w/env.bat.tmpl (hunk 4 above)
Comment (landam) [reply mswindows/osgeo4w/env.bat.tmpl]: Right, `libgeotiff.bat` is called by `o4w_env.bat`. Removed by f6d8b3c77

**Pull request merged**